securedrop/journalist_templates/admin_add_user.html

```diff
@@ -11,10 +11,16 @@ <h1 id="add-user-heading">Add User</h1>
     <div class="container flex-end">
       <div>
         <label for="username">{{ gettext('Username') }}</label>
-        {{ form.username() }}
+        {% set aria_attributes = {} %}
+        {% if form.username.errors %}
+        {% set _dummy = aria_attributes.update({'aria-describedby': 'username-errors', 'aria-invalid': 'true'}) %}
+        {% endif %}
+        {{ form.username(**aria_attributes) }}
         {% for error in form.username.errors %}
+        <span id="username-errors">
```

Comment:
Doesn't this need to be outside the {% for ... %} loop?

Reply:
Thanks for catching this!  Fixed (here and below) in b717eee2f4b751ea767610c1492be5b50c421b6a.